src/riscv-integration.adoc

```diff
@@ -1078,27 +1078,42 @@ memory addresses must match for the address to be valid:
 * For Sv48, bits [63:48] must equal bit 47
 * For Sv57, bits [63:57] must equal bit 56
 
-RISC-V permits that some CSRs, such as <<mtvec>> and <<mepc>> (see
-xref:CSR_exevectors[xrefstyle=short]), need not be able to hold all possible
-invalid addresses. Prior to writing these CSRs, implementations may convert an
+RISC-V permits that some CSRs holding addresses, such as <<mtvec>> and <<mepc>>
+(see xref:CSR_exevectors[xrefstyle=short]) and consequently <<pcc>>, need not be able to hold all possible
```

Comment:
Minor nit: `consequently` is a bit weird here. It isn't really a consequence of `mtvec` or `mepc` which is what it sounds like.

Maybe:

```
RISC-V permits that some registers that only hold addresses, such as <<mtvec>> and <<mepc>> (see ...) and also <<pcc>>, need not ...
```

Reply:
Done!